gam/gam.py

```diff
@@ -26,10 +36,12 @@ class GAM:
         k (int): number of subpopulations to surface
     """
 
-    def __init__(self, attributions_path="local_attributions.csv", distance="spearman", k=2):
+    def __init__(
+        self, attributions_path="local_attributions.csv", distance="spearman", k=2
+    ):
```

Comment:
I think I am going to setup a pylint.rc that can maybe adopt the black formatting in my next PR and we can establish some project linting standards. One thing that I like to do is increase the line limit to 120. I have found that the standard for PEP8 is a little too short and it sometimes leads to things being less readable. What do you think?

Reply:
Yeah, I think that's great. Definitely on board for building and maintaining standards. And I agree, 120 is restrictive to the point of unreadability at times.

Reply:
Sounds good - I'll have a pr up soon that we can drive some of these specifics to.